src/Main.java

```diff
@@ -3,17 +3,9 @@ public class Main {
 
     public static void main(String[] args) {
         try {
-            validation("Alexandr_02", "aB123456789", "aB123456789");
-        } catch (RuntimeException e) {
+            Service.check("Alexandr_02", "aB123456789", "aB123456789");
+        } catch (WrongLoginException | WrongPasswordException e) {
             System.out.println("Неверный логин или пароль");
```

Comment:
```suggestion
            System.out.println("Неверный логин или пароль: " + e.getMessage());
```
чтобы не терять причину, лучше вывести сообщение